TGs/verification-task-group/projects/CVA6/projects/cv32a6-v0.1.0-milestone/meetings/2023/cv32a6_v0.1.0_verif_KOM_minutes.md

```diff
@@ -1,62 +1,61 @@
-	b.	cv32a6_v0.1.0 project aims to develop verification environment, this environment will be applied on other cva6 version, cv32a6_v0.2.0 for instance.
-	c.	Do we need to work in core-v-verif/cva6/dev branches?
-	d.	cv32a6_v0.1.0 project aims to verify “software programmer view level”.
-	e.	Current CVA6 version is 4.2. Should we rename cv32a6_v0.1.0 into cv32a6_v4.3.0 for instance ?  For information, latest tag on cva6 RTL is 4.2.0
-	f.	cv32a6_v0.2.0 and cv32a6_v0.3.0 are not granted, but they are in the roadmap.
-	g.	cv32a6_v0.1.0 project consists to adapt the existing embedded cores verification infrastructure to cv32a6_v0.1.0: specifications, dvplans, UVM agents, …  New points to be verified should be kept to be verified during next steps, cv32a6_v0.2.0 for instance.
-	h.	Functional coverage goal is not 100%, to shorten cv32a6_v0.1.0 project and keep team motivation. 100% will be reached in following projects, cv32a6_v0.2.0 for instance.
+  * (a) cv32a6_v0.1.0  verification is part of CVA6 OpenHW group project. Jean-Roch Coulon is Verification technical leader and subproject manager. 
+  * (b)	cv32a6_v0.1.0 project aims to develop verification environment, this environment will be applied on other cva6 version, cv32a6_v0.2.0 for instance.
+  * (c)	Do we need to work in core-v-verif/cva6/dev branches?
+  * (d)	cv32a6_v0.1.0 project aims to verify “software programmer view level”.
+  * (e)	Current CVA6 version is 4.2. Should we rename cv32a6_v0.1.0 into cv32a6_v4.3.0 for instance ?  For information, latest tag on cva6 RTL is 4.2.0
+  * (f)	cv32a6_v0.2.0 and cv32a6_v0.3.0 are not granted, but they are in the roadmap.
+  * (g)	cv32a6_v0.1.0 project consists to adapt the existing embedded cores verification infrastructure to cv32a6_v0.1.0: specifications, dvplans, UVM agents, …  New points to be verified should be kept to be verified during next steps, cv32a6_v0.2.0 for instance.
+  * (h)	Functional coverage goal is not 100%, to shorten cv32a6_v0.1.0 project and keep team motivation. 100% will be reached in following projects, cv32a6_v0.2.0 for instance.
 
 2.	cv32a6_v0.1.0  plan
-	a.	Done in GANTT.
-	b.	Resources from Thales (TSS, TRT, ECC), OpenHW Group, ETHZ
-	c.	Newcomers: 10xEngineers, Bosch, Synthara,… contributions most welcome, especially to address gaps; get in touch with Jean-Roch.
-	d.	Not yet assigned tasks: MMU, A Extension, fencei, core_cntrl, linter, fix bugs, OVPSim
-	e.	Target completion date: September 2023
+  * (a) Done in GANTT.
+  * (b) Resources from Thales (TSS, TRT, ECC), OpenHW Group, ETHZ
+  * (c) Newcomers: 10xEngineers, Bosch, Synthara,… contributions most welcome, especially to address gaps; get in touch with Jean-Roch.
+  * (d) Not yet assigned tasks: MMU, A Extension, fencei, core_cntrl, linter, fix bugs, OVPSim
+  * (e) Target completion date: September 2023
 
-3.	For information, SAIL provides an help to generate instruction specification.
```

Comment:
Thanks for the reformat. The SAIL information has been removed ;-)

Reply:
Not removed, just _moved_.  It is now under point 4, "ISS".  It seemed like an orphan all by itself.